src/config/chains.ts

```diff
@@ -1,4 +1,22 @@
 import { networks } from '@safe-global/protocol-kit/dist/src/utils/eip-3770/config'
+import { type SafeVersion } from '@safe-global/safe-core-sdk-types'
+import { getSafeSingletonDeployment } from '@safe-global/protocol-kit/node_modules/@safe-global/safe-deployments'
+import semverSatisfies from 'semver/functions/satisfies'
+import { LATEST_SAFE_VERSION } from './constants'
+
+export const getLatestSafeVersion = (chainId?: string): SafeVersion => {
+  // Without version filter it will always return the LATEST_SAFE_VERSION constant to avoid automatically updating to the newest version if the deployments change
+  const latestVersion = (getSafeSingletonDeployment({ network: chainId, released: true })?.version ??
+    '1.3.0') as SafeVersion
```

Comment:
Let's extract `1.3.0` into a `FALLBACK_VERSION` constant.

Also, this file stopped being a "config" when we switched to generated chain lists. So I would suggest to not make it even less of a config and move this function to utils or services.